plugins/eddb.py

```diff
@@ -24,55 +24,68 @@
 
 
 import sys
+import tkinter
```

Comment:
Is this necessary given the conditional 'from' import below ?

Reply:
removed -- was added in an auto import while I was typing